.github/workflows/ossar-analysis.yml

```diff
@@ -1,20 +1,33 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
 # This workflow integrates a collection of open source static analysis tools
 # with GitHub code scanning. For documentation, or to provide feedback, visit
 # https://github.com/github/ossar-action
 name: OSSAR
 
 on:
   push:
+    branches: [ "master" ]
   pull_request:
```

Comment:
let's use parameters from example workflow at https://github.com/actions/starter-workflows/blob/main/code-scanning/ossar.yml:
```yml
on:
  push:
    branches: [ $default-branch, $protected-branches ]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: [ $default-branch ]
```

Reply:
Thank you - I have updated the GitHub action to use the given variables.

Reply:
I think I just need to update this as it's not correct to use the "$default-branch" value as they where just a placeholders for example if someone started to create OSSAR through github and this doesn't seem to apply on workflow files made manually.